dascore/io/dasdae/_compat.py

```
@@ -175,3 +175,47 @@ def translate_legacy_attrs(attrs, coord_names: Iterable[str] = ()):
+    with contextlib.suppress(
+        AttributeError,
+        EOFError,
+        ImportError,
+        IndexError,
+        KeyError,
+        pickle.PickleError,
+        TypeError,
+        UnicodeError,
+        ValueError,
+    ):
+        return pickle.loads(payload)
```

Ruff (0.16.2)

[error] `pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)